translations/translations.json

```diff
@@ -2401,20 +2401,8 @@
       "eventsDescription": "Aktuelle Veranstaltungen und Termine in der Umgebung",
       "offers": "Angebote",
       "offersDescription": "Jobsuche, Praktikasuche und mehr in der Umgebung",
-      "inquiryTitle": "Einstellungen",
-      "inquiryIntro": "Durch Klicken auf \"{{accept}}\" erlaube ich der {{appName}}-App, ...",
-      "sentryTitle": "App-Stabilität",
-      "sentryCondition": "... mit dem automatischen Senden von Absturzberichten {{appName}} zu verbessern",
-      "pushNewsTitle": "Push-Benachrichtigungen",
-      "pushNewsCondition": "... mir Push-Benachrichtigungen zu den neuesten lokalen Nachrichten zu senden",
       "skip": "Überspringen",
-      "next": "Weiter",
-      "cancel": "Abbrechen",
```

Comment:
Do we really want to delete these translations, its really generic stuff we may need sometime in the future?

Reply:
If we need them we can either translate them again or we look in the history and revert the commit. No need to keep them.